dashboards/apache-traffic-server/apache-traffic-server.json

```diff
@@ -0,0 +1,1583 @@
+						"width": 2,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT  latest(global.proxy.process.http.cache_deletes) AS 'cache deletes' FROM ATSCacheSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "HTTP Cache Lookups",
+					"layout": {
+						"column": 3,
+						"row": 1,
+						"width": 2,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.http.cache_lookups) AS 'cache lookups' FROM ATSCacheSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Total Cache Requests",
+					"layout": {
+						"column": 5,
+						"row": 1,
+						"width": 2,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.cache_total_requests) AS 'total cache requests' FROM ATSCacheSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Total Cache RAM",
+					"layout": {
+						"column": 7,
+						"row": 1,
+						"width": 3,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT latest(global.proxy.process.cache.ram_cache.total_bytes) / 1e+6 AS 'total cache RAM (MB)' FROM ATSCacheSampleMetrics"
+							}
+						],
+						"platformOptions": {
+							"ignoreTimeRange": false
+						}
+					}
+				},
+				{
+					"title": "Free Cache RAM",
+					"layout": {
+						"column": 10,
+						"row": 1,
+						"width": 3,
+						"height": 1
+					},
+					"linkedEntityGuids": null,
+					"visualization": {
+						"id": "viz.billboard"
+					},
+					"rawConfiguration": {
+						"facet": {
+							"showOtherSeries": false
+						},
+						"nrqlQueries": [
+							{
+								"accountIds": [],
+								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / 1e+6 AS 'free cache RAM (MB)' FROM ATSCacheSampleMetrics"
```

Comment:
```suggestion
								"query": "SELECT (latest(global.proxy.process.cache.ram_cache.total_bytes) - latest(global.proxy.process.cache.ram_cache.bytes_used)) / 1e+6 AS 'free RAM cache (MB)' FROM ATSCacheSampleMetrics"
```

Reply:
Hi @mdumpati,
As per your suggestions, I have made changes. Kindly review it.

Thanks.